core/include/usGetBundleContext.h

```diff
@@ -28,14 +28,15 @@
 #error Missing preprocessor define US_BUNDLE_NAME
 #endif
 
+#include <usBundleContext.h>
+#include <usBundleContextPrivate.h>
```

Comment:
Strange, I remember that I tried to find a solution to avoid having to include that header but didn't succeed. Now I looked at it again and it is actually quite simple. Good that you mentioned it!